### ISSUE TITLE
Unable to restart test with Tab + Enter

### ISSUE THREAD

**Issue by kennethcheo** [Miodec/monkeytype#1941]
Hi, not sure if anyone else is facing this or just me. 

![Animation](https://user-images.githubusercontent.com/42935499/135850373-715e0d71-d390-4e2a-8d86-c9b5b5cf8a51.gif)

I apologize if the gif isn't that clear.
- Pressing Tab selects the "Restart Test button"
- Pressing Enter unselects the button but the test doesn't restart. 
- It does work with the mouse though.

Expected behaviour: Tab + Enter should restart a test.

- OS: Tested on both Windows 10 and Linux (Xubuntu)
- Browser: Tested on Firefox 92.0.1, Chrome 94.0.4606.61 (normal and incognito mode)

I cleared my cache before this.

**Comment by Miodec** [comment]
Checked incognito?

**Comment by Miodec** [comment]
Deployed a small change. Be sure to clear cache and try again. If this doesn't work then I'm not sure, but you can work around the issue by enabling quick tab mode in the settings.

**Comment by SeerLite** [comment]
This one is weird as you're testing multiple OSs _and_ browsers and consistently getting the same issue, but none of us can reproduce it.

Are you using any extensions that could be messing with it?
Does it happen in the result screen too or only during the test?
Does it happen with no Monkeytype configuration?

**Comment by kennethcheo** [comment]
Still doesn't work after the PR, on incognito mode with my cache cleared and the default configuration. The only extensions I have are BitWarden and Ublock Origin, and I don't think they should mess with anything. 

I booted up my old laptop to test this out and it works there, so I thought it might be an issue with my PC/keyboard. But strangely enough, it works fine on the result screen? 

I enabled quick tab mode as a workaround. Feel free to close this issue if no one else is able to reproduce this. Thanks for your help!

**Comment by Miodec** [comment]
Maybe you have some external software that remaps keys or something? What happens when you press enter on this site? https://keycode.info/

**Comment by rachelsalazar** [comment]
I'm also having this error @kennethcheo. And @Miodec, I went to the keycode.info website and when I pressed enter it recognized the enter key.

**Comment by kennethcheo** [comment]
I managed to identify and fix the problem. I'm using QMK configurator to configure my Planck's layout.

I mapped the [`KC_SFTENT`](https://docs.qmk.fm/#/feature_space_cadet) key to use as my Enter key and it has worked normally on every use case so far. Even the key code returned by keycode.info recognizes it as an Enter key (when tapped). The gif recorder I used to capture the gif also recognizes it as Enter.

Remapping it back to a normal Enter key has fixed the problem. Perhaps someone would like to experiment with the different keycodes on QMK to understand why there this a difference?
